zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -712,6 +712,49 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
+  /process-instances/{processInstanceKey}/cancellation:
+    post:
+      tags:
+        - Process Instance
+      summary: Cancel process instance
+      description: Cancel a running process instance.
```

```suggestion
      description: Cancels a running process instance.
```
:wrench: Wording edit to standardise our API documentation. e.g. Summary: Cancel, Description: Cancels...